meta/autoid/autoid.go

```diff
@@ -941,3 +955,67 @@ func (l *AutoRandomIDLayout) IncrementalBitsCapacity() uint64 {
 func (l *AutoRandomIDLayout) IncrementalMask() int64 {
 	return (1 << l.IncrementalBits) - 1
 }
+
+// GetSequenceBackUpBase implements autoid.Allocator GetSequenceBackUpBase interface.
+// This function is used for backup with sequence setval.
+func (alloc *allocator) GetSequenceBackUpBase(tableID int64) (int64, error) {
```

Comment:
How is it different from `allocator.NextGlobalAutoID`?

Reply:
actually, it's quite same.
NextGlobalAutoID will return next applicable id, the upper use it directly, or the br rebase auto_increment with (id-1).

For sequence rebase/backup, the only way is via setval. the base parameter in setval() means the (id) is already been used, and next value will skip it and find next value.

Actually they(BR) can use NextGloalAutoID directly, and minus one for increment > 0 or plus one for increment < 0 cases cause NextGlobalAutoID will always return (id+1) regradless sequence feature.